playground/mocks/challenge.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  path: '/challenge',
+  method: 'POST',
+  render: (req, res) => {
+    res.append('Access-Control-Allow-Origin', '*');
```

Comment:
looks like dyson support cors by default which implies you dont need those `res.append...`.
could you double-check?

Reply:
You are correct.
<img width="1155" alt="Screen Shot 2019-12-06 at 2 30 23 PM" src="https://user-images.githubusercontent.com/5066836/70361012-0a198780-1835-11ea-858e-97fd4c72f919.png">